.github/workflows/deploy-docs.yml

```diff
@@ -52,10 +47,14 @@ jobs:
 
   # Deploy the documentation website
   deploy:
+    if: github.ref == 'refs/heads/main' && github.repository == 'OpenDevin/OpenDevin'
     name: Deploy to GitHub Pages
     runs-on: ubuntu-latest
+    # This job only runs on "main" so only run one of these jobs at a time
+    # otherwise it will fail if one is already running
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Should this have the same cancel in progress logic that existed previously?

Reply:
Not really. If this job is in progress, we should just let it finish instead of cancelling it.